src/Slider/Slider.js

```diff
@@ -313,9 +313,13 @@ Slider.propTypes = {
   domain: PropTypes.array,
   /**
    * An array of numbers. You can supply one for a value slider, two for a range slider or more to create n-handled sliders.
-   * The default values should correspond to valid step values in the domain.
+   * The values should correspond to valid step values in the domain.
    * The numbers will be forced into the domain if they are two small or large.
    */
+  values: PropTypes.array,
+  /**
+   * [DEPRECATED]: use `values` instead, which will also allow you to update the values from the outside.
+   */
```

Comment:
The idea would be to stop supporting `defaultValues` in the next major version. It's just a suggestion, of course.